mm2src/coins/tendermint/tendermint_coin.rs

```diff
@@ -1149,10 +1154,10 @@ impl TendermintCoin {
     }
 
     #[allow(clippy::result_large_err)]
-    pub(super) fn account_id_and_pk_for_withdraw(
+    pub(super) fn extract_account_id_and_private_key(
```

Comment:
You can remove `#[allow(clippy::result_large_err)]` annotation after changing err type to String